pallets/ddc-staking/src/lib.rs

```diff
@@ -504,6 +476,35 @@ pub mod pallet {
 			}
 			Ok(())
 		}
+
+		/// Update the DDC staking configurations .
+		///
+		/// * `bond_size`: The active bond needed to be a Storage or Edge node.
+		///
+		/// RuntimeOrigin must be Root to call this function.
+		///
+		/// NOTE: Existing nominators and validators will not be affected by this update.
+		#[pallet::weight(10_000)]
+		pub fn set_staking_configs(
+			origin: OriginFor<T>,
+			bond_size: ConfigOp<BalanceOf<T>>,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+
+			macro_rules! config_op_exp {
+				($storage:ty, $op:ident) => {
+					match $op {
+						ConfigOp::Noop => (),
```

Comment:
what's the use case for Noop?

Reply:
I take it from `pallet-staking` where they use it to allow partial config update. We have only one parameter now and preserving `Noop` looks redundant, but it's highly likely the `pallet-ddc-staking` will grow with more parameters, and preserving `Noop` here looks reasonable for future use.

Reply:
Ah ok